docs/pre-release/reference/config.md

```diff
@@ -46,6 +58,16 @@ These are the valid fields for the `logLevels` key:
 |`userDaemon`|Logging level to be used by the User Daemon (logs to connector.log)|debug|
 |`rootDaemon`|Logging level to be used for the Root Daemon (logs to daemon.log)|info|
 
+## Environment Variables
+Telepresence also can use Environment Variables to configure behavior.  Below is a chart listing
+the Environment Variables and which config key they map to (if applicable).  If both a config are
+specified, the Environment Variable will take precedence.
+
+|Environment Variable|Config Key|
```

Comment:
I only added configuration for the two env vars that people will realistically set, but should we include all of them in the docs and/or into config.yml? (https://github.com/telepresenceio/telepresence/blob/release/v2/pkg/client/envconfig.go#L15-L26)

Reply:
Ignore this since I'm going to remove this section.